frontend/libs/studio-pure-functions/src/NumberUtils/NumberUtils.ts

```diff
@@ -0,0 +1,6 @@
+export class NumberUtils {
+  /** Verifies if a number is within an open range (a range of which the minimum and maximum values are excluded) */
+  public static isWithinOpenRange = (value: number, min: number, max: number): boolean => {
```

Comment:
One more thing, what about renaming this method to `isBetweenRange`? 🤔

Reply:
Good idea to gather the range in one object. What about this? `range: { min: number, max: number }`
Then it is always clear that it is an object with two values, and what the role of each of them is. I'm not sure if an array makes sense here since arrays are usually meant for lists of which all values have the same role (yes, I know that React's `useState` hook is an exception to this). The syntax may also look confusing since this is how _closed_ intervals are defined in other programming languages (and mathematical notation in general), and this is an open one.

I thought of just naming it `isBetween`, but I thought it would be more precise to use the term "open" to differentiate it from closed and half-open intervals. I see that "interval" is a more mathamatically correct term though.